webapp/src/simulator.ts

```diff
@@ -329,7 +329,8 @@ export function run(pkg: pxt.MainPackage, debug: boolean,
         autoRun,
         ipc: isIpcRenderer,
         dependencies,
-        activePlayer: playerNumber
+        activePlayer: playerNumber,
```

Comment:
Couple quesitons
1. Are there any special things that we might want to do with custom themes and multiplayer selection to make sure that they work well together?
2. Will one setting override the other? For example, if someone is trying to make a multiplayer game with a theme that has the simulator be a different color and have a different configuration, will the theme make it so the multiplayer selection is not applied? Are there edge cases here where if you select a specific active player and then apply a theme the theme might not apply?

Reply:
Oh, I see my second question was answered in your PR comment here:
> Similarly, I left off any handling for displaying which user you are playing as in the sim when a user sets there skin & edits a multiplayer project; on the sim side I have it set that explicitly defined skin overrides the player # based one to ignore this for now.

Sorry about that!

Reply:
Good question to have though! And yeah, for next release we'll probably want to consider it more carefully (e.g. maybe dedicate a little area to indicate which player you are if the theme is explicitly set) but for now I just left it as theme field taking priority